backend/variables/builtin/readFile.js

```diff
@@ -12,19 +12,36 @@ const model = {
     definition: {
         handle: "readFile",
         usage: "readFile[\"path/to/file.txt\"]",
-        description: "Read contents of a text file. Optionally include 'true' as a second argument to select a random line.",
+        description: "Read contents of a text file. Optionally include 'true' as a second argument to select a random line, or a number to read a specific line.",
         possibleDataOutput: [OutputDataType.TEXT]
     },
     evaluator: (_, filePath, randomLine) => {
 
         if (filePath === null || !filePath.endsWith(".txt")) return "[File Path Error]";
         try {
             let contents = fs.readFileSync(filePath, "utf8");
-            if (randomLine) {
+            let filteredLines = [];
+
+            if (randomLine === "true" || !isNaN(randomLine)) {
                 let lines = contents.replace(/\r\n/g, "\n").split("\n");
-                let randIndex = util.getRandomInt(0, lines.length - 1);
-                return lines[randIndex];
+                filteredLines = lines.filter(function (line) {
+                    if (line != null && line.trim() !== "") {
+                        return line;
+                    }
+                });
+            }
+
+            // Get line at random.
+            if (randomLine === "true") {
```

Comment:
Use new `shouldReadRandomLine` variable suggested above:
```suggestion
            if (shouldReadRandomLine ) {
```